packages/salesforcedx-utils-vscode/src/commands/commandletExecutors.ts

```diff
@@ -117,6 +117,8 @@ export abstract class SfdxCommandletExecutor<T>
 
 export abstract class LibraryCommandletExecutor<T>
   implements CommandletExecutor<T> {
+  protected cancellable: boolean = false;
+  protected cancelled: boolean = false;
```

Comment:
```suggestion
  private cancelled: boolean = false;
```

Might not want concrete classes to be able to change this value, they should go through the token.

Reply:
good call!